app/logic/Calendar/ActiveSync/ActiveSyncEvent.ts

```diff
@@ -72,7 +72,10 @@ export class ActiveSyncEvent extends Event {
     this.location = sanitize.nonemptystring(wbxmljs.Location, "");
     this.isCancelled = (sanitize.integer(wbxmljs.MeetingStatus, 0) & 4) !== 0;
     let attendees = ensureArray(wbxmljs.Attendees?.Attendee);
-    if (wbxmljs.OrganizerEmail) {
+    if (wbxmljs.OrganizerEmail && attendees) {
```

Comment:
`&& attendees` is always going to be `true`, because you just ensured that it exists, on the line before. If you need the check, then say `&& attendees.length`.

Reply:
```suggestion
    if (wbxmljs.OrganizerEmail && attendees.length) {
```